support-workers/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -89,10 +88,10 @@ class SendThankYouEmail(thankYouEmailService: EmailService, servicesProvider: Se
           productOptions = p.productOptions,
           billingPeriod = p.billingPeriod,
           user = state.user,
-          paymentSchedule = state.paymentSchedule,
+          paymentSchedule = state.paymentMethod.left.toOption.map(_._2).getOrElse(PaymentSchedule(List())),
```

Comment:
not sure if I'm a massive fan of the _1 and _2 everywhere as it doesn't read well, but we get the built in logic for free.  Could create a custom case class plus codec, then also the json state would be more clear.